### ISSUE TITLE
Command center - make it leaner?

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#194313]
Love the command center, but my biggest issue is the height in pixels. It just takes too much of my screen real estate.
Could we save a couple of pixels to make it look lean? This way the title area looks too bloated compared to other apps.

@hediet had similar concerns

fyi @hbons @joaomoreno 

![Screenshot 2023-09-27 at 17 28 21](https://github.com/microsoft/vscode/assets/1926584/8cdbbe68-09d2-406a-9f1b-becf3b377bd7)

**Comment by jrieken** [comment]
Open for suggestion. It uses the default 35px that we use for view and editor titled. Without CC it's 28px.

**Comment by bpasero** [comment]
Maybe this would be a candidate for the new `window.density` namespace and offer a setting. With `window.density.editorTabHeight` you can set to `compact` to reduce tab height to `22px`.

Later the idea would be that a user could set `window.density: compact` and all associated components would respect that.

**Comment by isidorn** [comment]
I hacked some css and decreased width to 28px and I think it looks decent. There are probably bugs I am missing.
So I am not sure if this really needs the extra space we give it?

@bpasero I think that is a fine solution if we do not find a good default.

![Screenshot 2023-09-27 at 18 08 51](https://github.com/microsoft/vscode/assets/1926584/e86abad6-17c2-45e1-b7db-4f9e663a77db)

**Comment by bpasero** [comment]
Yeah I am all up for reducing the default height, though `35px` is a good default we use elsewhere. Then we have `22px` for e.g. status bar.

If the compact version would be a small as `22px` (not sure thats possible), then we could do that behind a setting.

**Comment by gjsjohnmurray** [comment]
Please be aware that the pixels below the CC act as a grab handle for dragging the window, given that we can't (yet?) drag it from within the CC itself.

**Comment by isidorn** [comment]
@gjsjohnmurray thanks. But area left and right of the cc can still be used to drag. So I am not very concerned about this.

**Comment by gjsjohnmurray** [comment]
@isidorn you're on Mac, and for Windows users it's sometimes a bit limited:

![image](https://github.com/microsoft/vscode/assets/6726799/5c8dbe07-5d26-4aec-95dd-710b086fc96e)

**Comment by jrieken** [comment]
22px might be too aggressive btw. It would require us to shrink all buttons and menu items more than in other places

<img width="1777" alt="Screenshot 2023-09-27 at 18 13 59" src="https://github.com/microsoft/vscode/assets/1794099/ff34f2a6-13a3-4533-876e-725670b5961c">

**Comment by isidorn** [comment]
@jrieken what do you think about 28px? 
@gjsjohnmurray good point we should check all platforms.

**Comment by abhijit-chikane** [comment]
#152578 I think the main reason you kept it to 35px is discuss here in this issue

**Comment by DerekZiemba** [comment]
35px would be alright if it didn't increase to 42px when `"window.zoomLevel": -.5`.  The titlebar doesn't scale with everything else. 

![image](https://github.com/microsoft/vscode/assets/7283693/ea9fecfa-ea9b-49bb-ae51-20ef1074ec10)
